test/UrlHelperTest.php

```diff
@@ -48,7 +48,7 @@ public function testRaisesExceptionOnInvocationIfNoRouteProvidedAndResultIndicat
 
     public function testRaisesExceptionOnInvocationIfRouterCannotGenerateUriForRouteProvided()
     {
-        $this->router->generateUri('foo', [])->willThrow(RouterException::class);
+        $this->router->generateUri('foo', [], [])->willThrow(RouterException::class);
```

Comment:
@weierophinney actually, we do need to change the tests. The argument is optional and defaults to `[]`, which eventually gets passed to `generateUri`. If we don't update the prophecies to expect that new `[]` param, it will fail:

e.g.

```
2) ZendTest\Expressive\Helper\UrlHelperTest::testRaisesExceptionOnInvocationIfRouterCannotGenerateUriForRouteProvided
Failed asserting that exception of type "Prophecy\Exception\Call\UnexpectedCallException" matches expected exception "Zend\Expressive\Router\Exception\RuntimeException". Message was: "Method call:
  - generateUri("foo", [])
on Double\RouterInterface\P38 was not expected, expected calls were:
  - generateUri(exact("foo"), exact([]), exact([]))" at
```